lib/improver/utilities/temporal.py

```diff
@@ -525,3 +526,51 @@ def process(self, cube_t0, cube_t1):
             interpolated_cubes.append(single_time)
 
         return interpolated_cubes
+
+
+def unify_forecast_reference_time(cubes, cycletime):
+    """Function to unify the forecast_reference_time across the input cubes
+    provided. The cycletime specified is used as the forecast_reference_time.
+
+    Args:
+        cubes (iris.cube.CubeList):
+            Cubes that will have their forecast_reference_time unified.
```

Comment:
Add "Any bounds on the forecast_reference_time coord will be discarded"

Reply:
Added.